assets/src/components/button/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Muriel-styled buttons.
+ */
+
+/**
+ * WordPress dependencies.
+ */
+import { Component } from '@wordpress/element';
+import { Button } from '@wordpress/components';
+
+class NewspackButton extends Component {
+
+	/**
+	 * Render.
+	 */
+	render( props ) {
+		const { value } = this.props;
+		return (
+			<div className="newspack-button">
```

Comment:
Is there any reason we need a wrapper, as opposed to simply adding the class to the `Button` component, like this?

`<Button className="newspack-button" { ...this.props } />`

Reply:
Nope, I think I just started out with this for some reason and it got left over.

Reply:
Yep, this is necessary in order to align the button properly when within a card component, hence [this code](https://github.com/Automattic/newspack-plugin/pull/46/files#diff-050da08994b8f727f91b4b19128cd8bdR73)